### ISSUE TITLE
See my own avatar

### ISSUE THREAD

**Issue by montage125** [matrix-org/thirdroom#385]
1. I hope to see my own avatar from the perspective of a third person, which makes me feel more aggressive.
2. I hope to integrate readyplayer Me avatar. Users can set their own avatars.

**Comment by spaetz** [comment]
Moving to a 3rd party view is possible now, isn't it?

**Comment by pablo-mayrgundter** [comment]
Yes, was working earlier this week for me, tho broken for me today.  I found this bug looking for related issues
